src/Svelte/doc/index.rst

```diff
@@ -51,6 +51,19 @@ Next, install a package to help Svelte:
 That's it! Any files inside ``assets/svelte/controllers/`` can now be rendered as
 Svelte components.
 
+To make internal functions work in Svelte, such as ``onMount``, ``onDestroy``, etc,
```

Comment:
Could you add a link to the documentation you mention?

Reply:
Yes, sure! I got this link: 
https://svelte.dev/docs/v4-migration-guide#browser-conditions-for-bundlers

Should I add it to the documentation?

Reply:
Great. I will do it tomorrow. 👍